src/serac/physics/solid_mechanics.hpp

```diff
@@ -801,6 +801,55 @@ class SolidMechanics<order, dim, Parameters<parameter_space...>, std::integer_se
                                  mesh_, qdata);
   }
 
+  /**
+   * @brief Functor representing a material stress.  A functor is used here instead of an
+   * extended, generic lambda for compatibility with NVCC.
+   */
+  template <typename Material>
+  struct SetMaterialStressFunctor {
+    /// @brief Constructor for the functor
+    SetMaterialStressFunctor(Material material, GeometricNonlinearities gn) : material_(material), geom_nonlin(gn) {}
+    /// @brief Material model
+    Material material_;
+    /// @brief Enum value for geometric nonlinearities
+    GeometricNonlinearities geom_nonlin;
```

Comment:
Member variables should have a trailing underscore
```suggestion
    GeometricNonlinearities geom_nonlin_;
```